test/e2e/tests/grpc_test.go

```diff
@@ -36,31 +36,28 @@ func TestGRPC_Version(t *testing.T) {
 func TestGRPC_Block_GetByHeight(t *testing.T) {
 	testFullNodesOrValidators(t, 0, func(t *testing.T, node e2e.Node) {
 		t.Helper()
-		client, err := node.Client()
-		require.NoError(t, err)
-		status, err := client.Status(ctx)
-		require.NoError(t, err)
-
-		// We are not testing getting the first block in these
-		// tests to prevent race conditions with the pruning mechanism
-		// that might make the tests fail. Just testing the last block
-		// is enough to validate the fact that we can fetch a block using
-		// the gRPC endpoint
-		last := status.SyncInfo.LatestBlockHeight
 
 		ctx, ctxCancel := context.WithTimeout(context.Background(), time.Minute)
 		defer ctxCancel()
 		gRPCClient, err := node.GRPCClient(ctx)
 		require.NoError(t, err)
 		defer gRPCClient.Close()
 
+		latestHeightCh, err := gRPCClient.GetLatestHeight(ctx)
+		require.NoError(t, err)
+
+		latest, ok := <-latestHeightCh
+		require.Equal(t, ok, true)
+		require.Greater(t, latest.Height, int64(0))
```

Comment:
Can we make this stronger? Greater than 0 seems obvious here, since we don't have block 0, eheh.

Reply:
addressed on commit 899e1f486eb4413dbec570ae69ee8858d37e3a5c